whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapRelationTypeConverter.java

```diff
@@ -0,0 +1,34 @@
+package net.ripe.db.whois.api.rdap;
+
+import jakarta.ws.rs.ext.ParamConverter;
+import jakarta.ws.rs.ext.ParamConverterProvider;
+import jakarta.ws.rs.ext.Provider;
+import net.ripe.db.whois.api.rdap.domain.RelationType;
+import org.springframework.stereotype.Component;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+
+@Component
+@Provider
+public class RdapRelationTypeConverter implements ParamConverterProvider {
+
+    @Override
+    public <T> ParamConverter<T> getConverter(Class<T> rawType, Type type, Annotation[] annotations) {
+        if(!rawType.equals(RelationType.class)) {
+            return null;
+        }
+
+        return (ParamConverter<T>) new ParamConverter<RelationType>() {
+            @Override
+            public RelationType fromString(String relationType){
```

Comment:
is it possible to add a toString method in enum for this ?

Reply:
check RdapRequestTypeConverter
I am following the same logic.

so I suggest leaving in this way for consistency. Maybe we can change it all together?

Reply:
No, we are creating a new one so we should do it correctly.

Reply:
OK, I moved into the enum

Reply:
done